components/raftstore/src/store/peer.rs

```diff
@@ -3955,8 +3953,8 @@ mod tests {
             ProposalQueue::new("tag".to_owned());
         let t = monotonic_raw_now();
         let gen_term = |index: u64| (index / 10) + 1;
-        for index in 1..=100 {
-            let renew_lease_time = if index % 3 == 1 { None } else { Some(t) };
+        let push_proposal = |pq: &mut ProposalQueue<_>, index: u64| {
+            let renew_lease_time = if index % 3 == 0 { None } else { Some(t) };
```

Comment:
How about using different values?

Reply:
I use `u64_to_timespec` to get a unique propose time for each proposal instead, PTAL